packages/block-library/src/navigation/index.php

```diff
@@ -77,12 +77,12 @@ function render_block_navigation( $attributes, $content, $block ) {
 function build_navigation_html( $block, $colors ) {
 	$html            = '';
 	$css_classes     = implode( ' ', $colors['css_classes'] );
-	$class_attribute = sprintf( ' class="%s"', esc_attr( ! empty( $css_classes ) ? 'wp-block-navigation-item__link ' . $css_classes : 'wp-block-navigation-item__link' ) );
+	$class_attribute = sprintf( ' class="wp-block-navigation-link__content %s"', esc_attr( trim( $css_classes ) ) );
 	$style_attribute = $colors['inline_styles'] ? sprintf( ' style="%s"', esc_attr( $colors['inline_styles'] ) ) : '';
 
 	foreach ( (array) $block['innerBlocks'] as $key => $block ) {
 
-		$html .= '<li class="wp-block-navigation-item">' .
+		$html .= '<li class="wp-block-navigation-link">' .
```

Comment:
I think we should consider there is only one _type of item_ available for the `<Navigation />` which is the `<NavigationLink />`, but it's possible in the near future that we will have more items, such as media items, a simple paragraph, etc.
For this, the name `link` should be applied only in the navigation item block, not in the navigation one, imo.

Reply:
@retrofox, the way the navigation block works is different to other blocks. The inner Navigation Link blocks are rendered server side by Navigation Block. This is the code that does that, so each `<li/>` here is a Navigation Link.

Maybe there's some way to improve this in the future so that the class name isn't defined in the parent block, but for now the main thing is that the editor and post class names match up as much as possible for themers.

Reply:
yes, I forgot and got confused a little bit. But my point is that the class name shouldn't be `link` for all items. I think we are completely ok so far since there is only one kind of item which is the link, but we should keep in mind that the idea is maybe to get more items. In fact, we could deduce the kind of item through the block attributes.